maqs-utilities/src/main/java/com/cognizantsoftvision/maqs/utilities/logging/IHtmlFileLogger.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 (C) Cognizant SoftVision, All rights Reserved
+ */
+
+package com.cognizantsoftvision.maqs.utilities.logging;
+
+/**
+ * The Html File Logger interface class.
+ */
+public interface IHtmlFileLogger extends AutoCloseable {
```

Comment:
Why doesn't this also have ILogger  added like we have for the file logger interface?